src/multiparser.js

```diff
@@ -37,26 +37,30 @@ function fromBabylonFlowOrTypeScript(path) {
        * ```
        */
       if (
-        parentParentParent &&
-        parent.quasis &&
-        parent.quasis.length === 1 &&
-        parentParent.type === "JSXExpressionContainer" &&
-        parentParentParent.type === "JSXElement" &&
-        parentParentParent.openingElement.name.name === "style" &&
-        parentParentParent.openingElement.attributes.some(
+        parentParent &&
+        node.quasis &&
+        parent.type === "JSXExpressionContainer" &&
+        parentParent.type === "JSXElement" &&
+        parentParent.openingElement.name.name === "style" &&
+        parentParent.openingElement.attributes.some(
           attribute => attribute.name.name === "jsx"
         )
       ) {
+        // Get full template literal with expressions replaced by placeholders
+        const rawQuasis = node.quasis.map(q => q.value.raw);
+        const text = rawQuasis.join("@prettier-placeholder");
+
         return {
           parser: "postcss",
```

Comment:
We should make all the CSS go through the same codepath instead of duplicating this non trivial logic.